dashboard/test/models/contact_rollups_processed_test.rb

```diff
@@ -112,24 +112,42 @@ class ContactRollupsProcessedTest < ActiveSupport::TestCase
     time_str = '2020-03-11 15:01:26'
     time_parsed = Time.find_zone('UTC').parse(time_str)
 
+    format_values = {
+      sources_key: ContactRollupsProcessed::SOURCES_KEY,
+      data_key: ContactRollupsProcessed::DATA_KEY,
+      data_updated_at_key: ContactRollupsProcessed::DATA_UPDATED_AT_KEY,
+      time_str: time_str
+    }
+
+    # Test inputs are JSON strings.
+    # Use string +format+ method so we don't have to escape every double quotes.
     tests = [
       {
-        input: format('[{"sources": "table1", "data": null, "data_updated_at": "%s"}]', time_str),
+        input: format(
```

Comment:
Could you put a plain text example of what the input for one of these tests ends up looking like? Requires some mental strain to figure out what this input ends up actually looking like.

Reply:
Done.